penguin/integrations/mcp/server_tools/blueprints.py

```
@@ -0,0 +1,370 @@
+            "created": created,
+            "updated": updated,
+            "skipped": skipped,
+            "total_items": len(blueprint.items),
+        },
+    }
+    if project is not None:
+        payload["project"] = serialize_project_payload(project)
+    return payload
+
+
+def _parse_blueprint(arguments: dict[str, Any]):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_parse_blueprint`

(ANN202)